packages/plugin-split-editing/src/codemirror.ts

```diff
@@ -22,6 +22,7 @@ import {
   rectangularSelection,
 } from '@codemirror/view'
 import { $ctx } from '@milkdown/utils'
+import { StyleModule } from 'style-mod'
```

Comment:
Introduces a new package but doesn't add it to package.json, it makes [test failed](https://github.com/enpitsuLin/milkdown-lab/actions/runs/10267339426/job/28439570473?pr=124#step:7:42)

Reply:
use [pnpm](https://pnpm.io/) first pls, then run 
```sh
pnpm -C packages/plugin-split-editing add style-mod
```

Reply:
as i said previously, I cannot build the library on my machine (i get several errors related to the dev env). I also use pnpm on my project, maybe there is a conflict. But in any case, I have tested the code in other ways.
If a package is missing in package.json, I guess you can also add it yourself, no? As this is your project and I allowed you to make changes to the PR. You can also fork the PR's branch and finish the work on your end... 
I am already very kind to submit a PR with bug fix instead of filling an issue and waiting for the solution to come by itself.

Reply:
👌🏻I got u, let me help, so merge this now